IPython/core/shellapp.py

```diff
@@ -7,6 +7,7 @@
 # Copyright (c) IPython Development Team.
 # Distributed under the terms of the Modified BSD License.
 
+from distutils.version import LooseVersion as V
```

Comment:
distutils is in the process of being removed from Python, let's not introduce new dependencies on it.

Reply:
Arg, I just saw that. Is there another stdlib version-parsing implementation?

Reply:
nvm, we have `traitlets.version_info`